docs/Quickstart.md

```diff
@@ -62,24 +61,23 @@ During the execution of serving deployment, Llumnix will:
 Following these steps, Llumnix acts as the request scheduling layer situated behind the multiple frontend API servers and above the multiple backend vLLM engine instances. This positioning allows Llumnix to significantly enhance serving performance through its dynamic, fine-grained, and KV-cache-aware request scheduling and rescheduling across instances.
 
 ## Ray Cluster Notice
-When you include the --launch-ray-cluster option in Llumnix's serving deployment command, Llumnix automatically builds a Ray cluster during the execution of serving deployment. This action will overwrite any existing Ray cluster. If this behavior is not desired, simply omit the --launch-ray-cluster option, and Llumnix will initiate its actor components within the current Ray cluster.
+When you set RAY.LAUNCH_CLUSTER to True, Llumnix automatically builds a Ray cluster during the execution of serving deployment. This action will overwrite any existing Ray cluster. If this behavior is not desired, simply omit the RAY.LAUNCH_CLUSTER option, and Llumnix will initiate its actor components within the current Ray cluster.
 
 
 # Benchmarking
 We provide a benchmarking example to help you get through the usage of Llumnix.
 First, you should start the server to launch Llumnix and backend LLM engine instances:
 ```
 HEAD_NODE=1 python -m llumnix.entrypoints.vllm.api_server \
-                --host $HOST \
-                --port $PORT \
-                --initial-instances $INITIAL_INSTANCES \
-                --launch-ray-cluster \
+                --config-file $CONFIG_PATH \
                 --model $MODEL_PATH \
                 --engine-use-ray \
                 --worker-use-ray \
-                --max-model-len 4096
+                --max-model-len 4096 \
+                MANAGER.INITIAL_INSTANCES 1 \
```

Comment:
why we two kinds of different parameter styles `--max-model-len` and `MANAGER.INITIAL_INSTANCES`? we should have the same style for CLI flags (it's ok to use another style in your config file, but do not mix two styles in CLI)

Reply:
In gereral, MANAGER.INITIAL_INSTANCES should not be set from cli. It should be set in config.yml. Here, just to give an example how to overwrite the parameter from cli.

Reply:
when use config_file, we start the servive by :

python -m llumnix.entrypoints.vllm.api_server \
    --config-file $CONFIG_PATH \
    # vLLM arguments ...
    # Llumnix arguments ...
    ...

Llumnix arguments mean MANAGER.INITIAL_INSTANCES ... and so on. But in gereral, MANAGER.INITIAL_INSTANCES should not be set from cli. It should be set in config.yml. Here, just to give an example how to overwrite the parameter from cli.

Reply:
My point is, all the cli parameters should have the same naming style. We should not use `MANAGER.INITIAL_INSTANCES` in cli, instead, use `--initial-instances`. We then assign the value to the corresponding parameter in the internal config class.

And, if you don't want users to set a parameter from CLI, then just delete this parameter from CLI. If you include the parameter in CLI, then use the CLI naming style.